src/ImageViewerCaptureTool.cpp

```diff
@@ -19,16 +19,19 @@ ImageViewerCaptureTool::ImageViewerCaptureTool(uint width, uint height) {
 }
 
 ImageViewerCaptureTool::ImageViewerCaptureTool(double fovY, double fovX, uint value, bool isHeight) {
-    double aspectRatio = fovX / fovY;
     uint width, height;
+    double fovYrad = fovY * M_PI / 180.0;
```

Comment:
Change. No degrees in Rock/FlatFish code. If some external APIs need degrees, you do the conversion when calling that API.

In general, never mix different representations of the same quantities. Angles in Rock are in radians, don't start having mixes of degrees and radians. No exceptions.